data/json/snippets/survivor_notes.json

```diff
@@ -687,6 +687,169 @@
       {
         "id": "note_253",
         "text": "\"G, There's something not right with the forests here.  The ground there feels soft and is overgrown with some sort of mushrooms.  Be careful not to breathe in the dust they kick up, I accidentally took a whiff and have been coughing and vomiting since.  The map shows there's a cabin just outside town, I'll lay low  there until I feel better.  I'm almost home.  I know you will be waiting for me there.\""
+      },
+      {
+        "id": "msn_note_01",
+        "text": "\"Killed my best friend yesterday, I can't believe the world has come to this.\""
+      },
+      {
+        "id": "msn_note_02",
+        "text": "\"Lindsay, meet us at Jason's house.  Jason, the nut that he is, is going to set it on fire at midnight before we all make a break for it.  He's convinced the monsters aren't afraid of fire, but DRAWN to it. If it works it's at least going to make one giant distraction.  Let's hope it'll clear the entire block.\""
+      },
+      {
+        "id": "msn_note_03",
+        "text": "\"Jason, get all the dry food out of the kitchen before you… burn everything.  And stuff your belly full of anything that's going to go bad soon. -Mom\""
+      },
+      {
+        "id": "msn_note_04",
+        "text": "\"…and remember: Do the math.  Your life depends on it.  If you have 30 rounds in a rifle, you can take down maybe 10 zombies with it if you're a good shot… and lucky.  Can you kill one of those things with 3 rounds?\""
+      },
+      {
+        "id": "msn_note_05",
+        "text": "\"Dad said something about a bear trap and a tear gas grenade.  I… I think he's going to trap and gas that freak moose out on the street.  It'll give us meat for weeks if we use the dehydrator.  He hasn't come back though…\""
+      },
+      {
+        "id": "msn_note_06",
+        "text": "\"Honey, we're going to hole up in the garage for a day before we make a break for it out of the city.  If we try to plow through the horde, I fear the car won't stand ten seconds.  I'm going to move the engine to the back, and maybe the fuel tank too… add some plating to the front… Maybe reinforce the windows.  The modifications are going to take at least a day.  Find my caffeine pills.  I may not get any sleep in the next 36 hours.\""
+      },
+      {
+        "id": "msn_note_07",
+        "text": "\"Remember to carry some disinfectant.  I ran out, a lighter and a knife had to do in a pinch.\""
```

Comment:
Didn't we remove cauterization?

Reply:
I suppose the note could have been written by a medically inept individual, who just thinks they disinfected their wound.

Reply:
From what I remember, cauterization was already broken due to a code error, so references to it in item descriptions were removed.